fl4health/server/base_server.py

```diff
@@ -164,27 +165,71 @@ def poll_clients_for_sample_counts(self, timeout: Optional[float]) -> List[int]:
 
         return sample_counts
 
-    def _split_metrics(
+    def _unpack_metrics(
         self, results: List[Tuple[ClientProxy, EvaluateRes]]
     ) -> Tuple[List[Tuple[ClientProxy, EvaluateRes]], List[Tuple[ClientProxy, EvaluateRes]]]:
         val_results = []
         test_results = []
 
         for client_proxy, eval_res in results:
-            val_metrics = {k: v for k, v in eval_res.metrics.items() if not k.startswith("test - ")}
-            test_metrics = {k: v for k, v in eval_res.metrics.items() if k.startswith("test - ")}
+            val_metrics = {
+                k: v for k, v in eval_res.metrics.items() if not k.startswith(TestMetricPrefix.TEST_PREFIX.value)
+            }
+            test_metrics = {
+                k: v for k, v in eval_res.metrics.items() if k.startswith(TestMetricPrefix.TEST_PREFIX.value)
+            }
 
-            # Remove loss and num_examples from test_metrics if they exist
-            test_loss = float(test_metrics.pop("test - loss"))
-            test_num_examples = int(test_metrics.pop("test - num_examples"))
+            if len(test_metrics) > 0:
+                assert (
+                    TestMetricPrefix.TEST_PREFIX.value + "loss" in test_metrics
+                    and TestMetricPrefix.TEST_PREFIX.value + "num_examples" in test_metrics
+                ), (
+                    TestMetricPrefix.TEST_PREFIX.value
```

Comment:
We use f-strings by convention for this type of thing. If you define the constant I suggested above, this would be
`f"'{TEST_NUM_EXAMPLES}' and '{TEST_LOSS}' keys must be present in test_metrics dictionary for aggregation`

Reply:
done!